src/test/java/net/minestom/server/entity/EntityVelocityIntegrationTest.java

```diff
@@ -102,6 +105,30 @@ public void doubleKnockback(Env env) {
         });
     }
 
+    @Test
+    public void testHasVelocity(Env env) {
+        var instance = env.createFlatInstance();
+        loadChunks(instance);
+
+        var entity = new Entity(EntityType.ZOMBIE);
+        // Should  be false because the new entity should have no velocity
+        assertFalse(entity.hasVelocity());
```

Comment:
Don't need this check anymore, since we're checking after the entity joins

Reply:
Yeah I didnt look at all the other checks.
But I dont really see the problem here: If the check fits in two places, why not have it in both?

Reply:
technically there isn't an issue, but there also isn't a point in checking twice since within this test the value cannot change.

Having two checks here would be like testing if setting the instance of an entity also sets the entity velocity, which is not the goal of this test. It's always better to test specific events and actions, instead of writing one test to check multiple events.

Reply:
makes sense, got rid of the check